### DISCUSSION TITLE
How to use Opus 1.4 exe?

### DISCUSSION THREAD

**Discussion by bozzis** [enzo1982/freac#532] [Help & Support]
Hello, I need to convert using Opus 1.4.
I have the opusenc.exe , so I tried copying it to the codecs/cmdline folder but fre:ac didn't recognize it.
Is there a way that I can make fre:ac use that opus 1.4 exe that I have?

**Comment by Lee-Carre** [comment]
> I need to convert using Opus 1.4.

Because? What be wrong with v1·3?

> I have the `opusenc.exe`, so I tried copying it to the codecs/cmdline folder but fre:ac didn't recognize it. Is there a way that I can make fre:ac use that opus 1.4 exe that I have?

Possibly relevant documentation: [Fre:AC § Documentation § \(Manual\|User Guide\) § How to …? § How to install encoder/decoder xy?](//www.freac.org/manual/en/howto.html#anchor_enc).

> How to use Opus 1.4[…]?

Considering [commit `14ec602` (“Update Opus to version 1.4.”)](/enzo1982/freac/commit/14ec602a8751b38c68827d4c8f569dd40affabe8) [from 2023-04-24], I would suggest trying the [continuous build](/enzo1982/freac/releases/tag/continuous) (generated from the latest Git source-code, so would include the cited commit). No need to futz with external libraries/binaries, then.
